tests/operators/inverse_mass_variable_coefficient.output

```diff
@@ -66,6 +66,74 @@
   consider_inverse_coefficient = 1   : || vec  - ref ||_infty = 4.85723e-15
 
 
+  element_type     = Simplex
```

Comment:
note how all combinations now give good results for any number of components and `&dst == &src`